File: lib/OnlineServices/MobileApplication.dart
import '../Car.dart';
import 'OnlineSystem.dart';


class MobileApplication {
  OnlineSystem onlineSystem;

  MobileApplication(this.onlineSystem);

  void requestCarServicing() {
    onlineSystem.requestCarServicing();
  }

  void requestCarWashing() {
    onlineSystem.requestCarWashing();
  }

  void requestOnlineDelivery() {
    onlineSystem.requestOnlineDelivery();
  }
}

File: src/OnlineServices/MobileApplication.java
package OnlineServices;
public class MobileApplication {
    private OnlineSystem onlineSystem;

    public MobileApplication(OnlineSystem onlineSystem) {
        this.onlineSystem = onlineSystem;
    }

    public void requestCarServicing() {
        onlineSystem.requestCarServicing();
    }

    public void requestCarWashing() {
        onlineSystem.requestCarWashing();
    }

    public void requestOnlineDelivery() {
        onlineSystem.requestOnlineDelivery();
    }
}
